Assets/Scripts/Data Model/Ecology/FishingDistributionDelta.cs

```diff
@@ -1,114 +1,114 @@
+		/// </summary>
+		public FishingDistributionDelta(List<FishingObject> objects)
+		{
+			LoadDistribution(objects);
+		}
 
-	public bool HasDistributionValues()
-	{
-		return values.Count > 0;
-	}
+		public Dictionary<int, float> FindValuesForFleet(string fleetName)
+		{
+			Dictionary<int, float> result;
+			values.TryGetValue(fleetName, out result);
+			return result;
+		}
 
-	public void Clear()
-	{
-		values.Clear();
-	}
+		public IEnumerable<KeyValuePair<string, Dictionary<int, float>>> GetValuesByFleet()
+		{
+			return values;
+		}
 
-	public void SetFishingValue(string fleetName, int country, float fishingValue)
-	{
-		Dictionary<int, float> fleetValues;
-		if (!values.TryGetValue(fleetName, out fleetValues))
+		public bool HasDistributionValues()
 		{
-			fleetValues = new Dictionary<int, float>(TeamManager.TeamCount);
-			values.Add(fleetName, fleetValues);
+			return values.Count > 0;
 		}
 
-		fleetValues[country] = fishingValue;
-	}
+		public void Clear()
+		{
+			values.Clear();
+		}
 
-	private void LoadDistribution(List<FishingObject> deltaValues)
-	{
-		values.Clear();
-		if (deltaValues != null)
+		public void SetFishingValue(string fleetName, int country, float fishingValue)
 		{
-			foreach (FishingObject obj in deltaValues)
+			Dictionary<int, float> fleetValues;
+			if (!values.TryGetValue(fleetName, out fleetValues))
 			{
-				SetFishingValue(obj.type, obj.country_id, obj.amount);
+				fleetValues = new Dictionary<int, float>(TeamManager.TeamCount);
+				values.Add(fleetName, fleetValues);
 			}
+
+			fleetValues[country] = fishingValue;
 		}
-	}
 
-	public void SubmitToServer(int planId, BatchRequest batch)
-	{
-		List<FishingObject> valuesToSubmit = new List<FishingObject>(32);
+		private void LoadDistribution(List<FishingObject> deltaValues)
+		{
+			values.Clear();
+			if (deltaValues != null)
+			{
+				foreach (FishingObject obj in deltaValues)
+				{
+					SetFishingValue(obj.type, obj.country_id, obj.amount);
+				}
+			}
+		}
 
-		foreach (KeyValuePair<string, Dictionary<int, float>> fishingType in values)
+		public void SubmitToServer(int planId, BatchRequest batch)
 		{
-			foreach (KeyValuePair<int, float> kvp in fishingType.Value)
+			List<FishingObject> valuesToSubmit = new List<FishingObject>(32);
+
+			foreach (KeyValuePair<string, Dictionary<int, float>> fishingType in values)
 			{
-				valuesToSubmit.Add(new FishingObject {
+				foreach (KeyValuePair<int, float> kvp in fishingType.Value)
+				{
+					valuesToSubmit.Add(new FishingObject {
 						country_id = kvp.Key,
 						type = fishingType.Key,
 						amount = kvp.Value
-				});
+					});
+				}
 			}
-		}
 
-		JObject dataObject = new JObject();
+			JObject dataObject = new JObject();
 
-		//form.AddField("fishing_values", valuesToSubmit);
-		dataObject.Add("plan", planId);
-		dataObject.Add("fishing_values", JToken.FromObject(valuesToSubmit));
+			//form.AddField("fishing_values", valuesToSubmit);
```

Comment:
Commented code, as always see if it makes sense to leave it there for the future, otherwise remove it! :)